client/state/data-layer/wpcom/sites/launch/index.js

```diff
@@ -30,6 +31,7 @@ const handleLaunchSiteRequest = dispatchRequest( {
 		receiveSite( data ),
 		updateSiteSettings( data.ID, data.options ),
 		requestSiteChecklist( data.ID ),
+		requestEligibility( data.ID ),
```

Comment:
It feels like this should belong inside the eligibility directories, but obviously since the rest of `onSuccess` is scattering out a bunch of requests this isn't a big deal.

Reply:
Ah true, I was being a bit lazy here. I do keep track of button busy state, so I could follow up in the component to fire off another action on success. I expect to 🔪 this code whenever private atomic sites are available, so I don't think this will live long.